### ISSUE TITLE
Møtedokumenter

### ISSUE THREAD

**Issue by sturtzel** [ks-no/gi-arkiv-2.0#57]
eInnsyn opererer med to typer dokumenter i tillegg til journalpost: Møteregistrering og Møtesaksregistrering.

Dette gjelder dokumenter knyttet til selve møtet (som innkalling og protokoll) og det gjelder dokumenter knyttet til den enkelte behandlingen (som saksprotokoll).

Noark 5 skiller ikke da den ikke har noen møte eller utvalgssaksreferanse til moeteregistering.

Vår arkivmelding støtter ingen av disse, men bør vel gjøre det.

**Comment by jarleborsheim** [comment]
Så er dette to nye typer med `registrering` som base? Tilsvarende som `journalpost`? Hvilke felter skal være med?

**Comment by sturtzel** [comment]
Ja. Sett det opp på dagens møte - blir neppe landet der, men vi kan ta prinsippet først.

**Comment by Jepresterud** [comment]
Dagens modell i eInnsyn bruker registreringene: møtedokumentregistrering, møtesaksregistrering og journalpost. Møtesaksregistrering har en referanse (referanseTilMøtesak) til saksfremlegget som journalpost. Møtedokumentregistrering har moetemappe som parent og inneholder kun referanse til dokumentbeskrivelse (som videre har referanse til dokumentobjekt).
Legger ved en forenkla skisse til modellen i eInnsyn
![einnsyn datamodell_tabell](https://user-images.githubusercontent.com/23337466/154483131-130e7d52-244c-42cd-adc2-4bd06d224469.png)
